.github/workflows/ci_packages.yml

```diff
@@ -60,4 +60,6 @@ jobs:
 
       - name: 'koch, Run CI'
         shell: bash
-        run: . ci/funs.sh && nimInternalBuildKochAndRunCI
+        run: |
+          export DYLD_LIBRARY_PATH="/usr/local/Cellar/openssl@1.1/1.1.1g/lib:$DYLD_LIBRARY_PATH"
```

Comment:
On M1 macs, it's not `1.1.1g`, I think it can't find the correct lib due to non-existence path.
```suggestion
          export DYLD_LIBRARY_PATH="/usr/local/Cellar/openssl@1.1/$(ls /usr/local/Cellar/openssl@1.1 | tail -n 1)/lib:$DYLD_LIBRARY_PATH"
```
<img alt="Screen Shot 2022-07-11 at 10 21 16 AM" src="https://user-images.githubusercontent.com/43804263/178185223-26dc7eed-fe7e-4078-8eeb-0c01f2d929cf.png">